nilearn/datasets/tests/test_func.py

```diff
@@ -170,20 +170,20 @@ def test_fetch_localizer_contrasts(tmp_path, request_mocker, localizer_mocker):
     assert not hasattr(dataset, 'tmaps')
     assert not hasattr(dataset, 'masks')
     assert isinstance(dataset.cmaps[0], str)
-    assert isinstance(dataset.ext_vars, np.recarray)
+    assert isinstance(dataset.ext_vars, pd.DataFrame)
     assert len(dataset.cmaps) == 2
-    assert dataset.ext_vars.size == 2
+    assert len(dataset['ext_vars']) == 2
```

Comment:
if you need to edit the test it means backwards compatibility is broken. It is possible that making the change is the best solution (eg we could decide that we always return pandas dataframes or series instead of numpy recarrays) but it needs to go through a deprecation cycle, and be discussed because it places a burden on users and so it can be done consistently for all fetchers